### PR TITLE
feat: Updated partial edit icons with new SVG icon

### PR DESCRIPTION
**Pull request by imnavanath**
### Description
- Updated with new SVG icon for previous edit icon

### Screenshots
- https://a.cl.ly/o0uEgPL5

### Types of changes
- Non breaking change

### How has this been tested?
- With customizer refresh

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Comment by premanshup** [comment]
@imnavanath Please resolve merge conflicts and check semantic pull request status.

**Comment by imnavanath** [comment]
> @imnavanath Please resolve merge conflicts and check semantic pull request status.

Done @premanshup

File: inc/assets/css/customizer-preview-rtl.css [hunk 1]
```diff
@@ -30,6 +37,17 @@
     border-radius: 2px;
 }
 
+body.customize-partial-edit-shortcuts-shown .customize-partial-edit-shortcut button {
+    width: 26px;
+    height: 26px;
+    min-width: 26px;
+    min-height: 26px;
+	border-width: 1px;
```
Comment (premanshup) [inline inc/assets/css/customizer-preview-rtl.css:45]: @imnavanath Check the indentation.

File: inc/customizer/extend-custom-controls/src/customizer.js [hunk 2]
```diff
@@ -748,6 +751,15 @@
 
 			});
 
+			setTimeout(function () {
+				let partialButtons = iframeBody.find( 'button.customize-partial-edit-shortcut-button' ),
+					image = $( '<span class="ahfb-svg-iconset ast-inline-flex svg-baseline"> <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 576 512"><path d="M402.6 83.2l90.2 90.2c3.8 3.8 3.8 10 0 13.8L274.4 405.6l-92.8 10.3c-12.4 1.4-22.9-9.1-21.5-21.5l10.3-92.8L388.8 83.2c3.8-3.8 10-3.8 13.8 0zm162-22.9l-48.8-48.8c-15.2-15.2-39.9-15.2-55.2 0l-35.4 35.4c-3.8 3.8-3.8 10 0 13.8l90.2 90.2c3.8 3.8 10 3.8 13.8 0l35.4-35.4c15.2-15.3 15.2-40 0-55.2zM384 346.2V448H64V128h229.8c3.2 0 6.2-1.3 8.5-3.5l40-40c7.6-7.6 2.2-20.5-8.5-20.5H48C21.5 64 0 85.5 0 112v352c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V306.2c0-10.7-12.9-16-20.5-8.5l-40 40c-2.2 2.3-3.5 5.3-3.5 8.5z"></path></svg> </span>' );
+
+				partialButtons.html( image );
+
```
Comment (premanshup) [inline inc/customizer/extend-custom-controls/src/customizer.js:759]:
```suggestion

```

File: inc/customizer/extend-custom-controls/src/customizer.js [hunk 3]
```diff
@@ -748,6 +751,15 @@
 
 			});
 
+			setTimeout(function () {
+				let partialButtons = iframeBody.find( 'button.customize-partial-edit-shortcut-button' ),
+					image = $( '<span class="ahfb-svg-iconset ast-inline-flex svg-baseline"> <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 576 512"><path d="M402.6 83.2l90.2 90.2c3.8 3.8 3.8 10 0 13.8L274.4 405.6l-92.8 10.3c-12.4 1.4-22.9-9.1-21.5-21.5l10.3-92.8L388.8 83.2c3.8-3.8 10-3.8 13.8 0zm162-22.9l-48.8-48.8c-15.2-15.2-39.9-15.2-55.2 0l-35.4 35.4c-3.8 3.8-3.8 10 0 13.8l90.2 90.2c3.8 3.8 10 3.8 13.8 0l35.4-35.4c15.2-15.3 15.2-40 0-55.2zM384 346.2V448H64V128h229.8c3.2 0 6.2-1.3 8.5-3.5l40-40c7.6-7.6 2.2-20.5-8.5-20.5H48C21.5 64 0 85.5 0 112v352c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V306.2c0-10.7-12.9-16-20.5-8.5l-40 40c-2.2 2.3-3.5 5.3-3.5 8.5z"></path></svg> </span>' );
+
```
Comment (premanshup) [inline inc/customizer/extend-custom-controls/src/customizer.js:757]:
```suggestion

```

File: inc/customizer/extend-custom-controls/src/customizer.js [hunk 4]
```diff
@@ -748,6 +751,15 @@
...
+				partialButtons.html( image );
+
+				iframeBody.removeClass('ast-partials-refreshing');
+			}, 100);
+
```
Comment (premanshup) [inline inc/customizer/extend-custom-controls/src/customizer.js:762]:
```suggestion

```

File: inc/customizer/extend-custom-controls/src/customizer.js [hunk 5]
```diff
@@ -665,6 +665,9 @@
 			AstCustomizerAPI.setDefaultControlContext();
 			astra_builder_clear_operation_session();
 
+			let iframeBody = $body.find('iframe').contents().find('body');
+
```
Comment (premanshup) [inline inc/customizer/extend-custom-controls/src/customizer.js:669]:
```suggestion

```

**Review by premanshup — CHANGES_REQUESTED** [review]

File: inc/assets/css/customizer-preview-rtl.css (hunk 1 above)
Comment (imnavanath) [reply inc/assets/css/customizer-preview-rtl.css:38]: Corrected 👍

File: inc/customizer/extend-custom-controls/src/customizer.js (hunk 5 above)
Comment (imnavanath) [reply inc/customizer/extend-custom-controls/src/customizer.js]: Reverted these changes as previous edit button was finalized.

File: inc/customizer/extend-custom-controls/src/customizer.js (hunk 4 above)
Comment (imnavanath) [reply inc/customizer/extend-custom-controls/src/customizer.js]: Reverted these changes as previous edit button was finalized.

File: inc/customizer/extend-custom-controls/src/customizer.js (hunk 3 above)
Comment (imnavanath) [reply inc/customizer/extend-custom-controls/src/customizer.js]: Reverted these changes as previous edit button was finalized.

File: inc/customizer/extend-custom-controls/src/customizer.js (hunk 2 above)
Comment (imnavanath) [reply inc/customizer/extend-custom-controls/src/customizer.js]: Reverted these changes as previous edit button was finalized.

**Review by premanshup — APPROVED** [review]

**Pull request merged**